### PR TITLE
add missing methods

### PR DESCRIPTION
**Pull request by hjanuschka**
adds missing methods from plugin template - as of https://github.com/fastlane/fastlane/issues/5469

edit: tests should work now, i hope i got all affected places 🐺

### CODE REVIEW

File: fastlane/lib/fastlane/plugins/plugin_info_collector.rb [hunk 1]
```diff
@@ -147,5 +148,24 @@ def collect_summary
...
       !summary.to_s.strip.empty?
     end
+    #
+    # Summary
+    #
+
+    def collect_details
+      details = nil
+      loop do
+        details = @ui.input("\nPlease enter a detailed information of this fastlane plugin:")
```
Comment (asfalcone) [inline fastlane/lib/fastlane/plugins/plugin_info_collector.rb]: How about "Please enter a detailed description of this fastlane plugin"?

**Comment by hjanuschka** [comment]
@asfalcone thx for the feedback - changes made 👍

**Comment by asfalcone** [comment]
@hjanuschka Thanks, found one more tiny thing, then I'll merge this 👍

**Comment by hjanuschka** [comment]
thx - changes made, via github ui - you wan't it squashed?

**Review by asfalcone — APPROVED** [review]

**Comment by asfalcone** [comment]
@hjanuschka thanks! No need to squash now that github has "Squash and merge" 😍

**Pull request merged**